Client/src/components/contanier/student/Apply/index.js

```diff
@@ -1,10 +1,29 @@
 import React, { Component } from 'react'
+import CourseCard from '../../../common/Course'
+import PickDay from '../../../common/PickDay'
+import Button from '../../../common/Button'
+import './index.css'
 
 export default class Apply extends Component {
+  state = {
+    course: {
+      id:0,
+      titel: 'Writing & Speaking',
+      desc: 'In this course you will learn the fundemtal of speaking and writign skills'
+    }
+  }
   render() {
+    const days = ["Pick a day ..", "Sat - Mon - Wedn", "Sun - Tues - Thurs"];
+    const hours = ["Pick a time ..", "8:00 - 10:00 PM", "10:00 - 12:00 PM", "12:00 - 14:00 AM"];
```

Comment:
can you put it in your state for easy read and to simplify your work when you fetch the data